package.json

```diff
@@ -55,5 +58,8 @@
     "ts-node": "^10.4.0",
     "typechain": "^5.2.0",
     "typescript": "^4.5.4"
+  },
+  "dependencies": {
+    "bee-factory": "github:ethersphere/bee-factory"
```

Comment:
ah, I see now. well, it is also a great solution, maybe even better than using git submodule, but if you changed this setup, then take out the github module dependency.